### PR TITLE
Rename "Filter content to "Hide content" in admin UI

### PR DESCRIPTION
**Pull request by aldenstpage**
## Fixes https://github.com/creativecommons/cccatalog-api/issues/561

### CODE REVIEW

File: cccatalog-api/cccatalog/api/models.py [hunk 1]
```diff
@@ -153,7 +153,9 @@ class ContentProvider(models.Model):
     provider_name = models.CharField(max_length=250, unique=True)
     created_on = models.DateTimeField(auto_now=False)
     domain_name = models.CharField(max_length=500)
-    filter_content = models.BooleanField(null=False, default=False)
+    filter_content = models.BooleanField(
+        null=False, default=False, verbose_name='Hide content'
+    )
```
Comment (zackkrida) [inline cccatalog-api/cccatalog/api/models.py:158]: 👍 happened to see this, looks good

**Comment by zackkrida** [comment]
@kgodey i may be wrong but i think alden kept the db field name the same and just changed the name in the ui to 'Hide content'.

**Comment by aldenstpage** [comment]
That's right

**Comment by kgodey** [comment]
I thought you needed migrations for any changes in the models, not just DB changes. I might be wrong but it's worth running `makemigrations` to check.

**Pull request merged**